deploy/docker-compose.yaml

```
@@ -69,6 +69,17 @@ services:
       interval: 5s
       timeout: 1s
       retries: 10
+  mongo-express:
+    image: mongo-express
+    environment:
+      - ME_CONFIG_MONGODB_SERVER=mongodb
+      - ME_CONFIG_MONGODB_PORT=27017
+      - ME_CONFIG_MONGODB_AUTH_USERNAME=admin
+      - ME_CONFIG_MONGODB_AUTH_PASSWORD=admin
+    depends_on:
+        - mongodb
+    ports:
+      - "8005:8081"
```

Nasze serwisy są na portach 8000, 8001, 8002 i 8003. Ja bym to mongoDB express zmienił na jakiś port typu `27000` bo samo mongo ma port `27017`
```suggestion
      - "27000:8081"
```